test/functional/terminal/tui_spec.lua

```diff
@@ -1469,6 +1469,33 @@ describe('TUI', function()
                                                                                  |
       {3:-- TERMINAL --}                                                             |]]))
   end)
+
+  it('visual bell (padding) does not crash #21610', function()
+    feed_data ':set visualbell\n'
```

Comment:
I think this test also need setting `belloff=`

Reply:
I don't think it does. it crashes on master.

Reply:
Oh, forgot that `nvim_argv` adds this by default.